users/me is deprecated and replaced by users/self (#143)
iZettle/api-documentation: CHANGELOG.adoc

- For API changes and the corresponding documentation changes, the log is tagged with the name of the changes API. For example, `OAuth API`.
- For documentation changes that are editorial and are not introduced by any API change, the log is tagged with `Editorial changes`.

.**January 18, 2022** `OAuth API` `Editorial changes`
[%collapsible]
====
**Replaced the deprecated endpoint `users/me` with `users/self`**

> The following updates are made for the deprecation:

Reviewer: ```suggestion
The following updates were done for the deprecation:
```

- The OAuth API endpoint `users/me` is deprecated and replaced by `users/self`.
- Until `users/me` is removed, all API requests to it will be redirected to `users/self`.
- The API documentation is updated to include `users/self` instead of `users/me`.

> > **Note:** The support for the deprecated `users/me` will continue until further notice. Please replace `users/me` to `users/self` in your integration as soon as possible.

Reviewer: ```suggestion
> **Note:** The support for the deprecated `users/me` will continue until further notice. It is recommended to replace `users/me` with `users/self` in your integration as soon as possible.
```
